src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const tabSelected = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      {tabs.length !== 0 && (
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                key={tab.id}
+                className={tab.id === tabSelected.id ? 'is-active' : ''}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() =>
+                    tab.id !== activeTabId ? onTabSelected(tab.id) : ''
```

Comment:
Would be great to create a variable for boolean condition as you can use the same in both cases. Also, consider using && instead of the ternary operator